spring-boot-starter/starter/pom.xml

```diff
@@ -149,7 +149,6 @@
     <dependency>
       <groupId>com.sun.xml.bind</groupId>
       <artifactId>jaxb-impl</artifactId>
-      <scope>test</scope>
```

Comment:
With this change this library is included in Camunda Run and we ship it (and I think we did not do that before). Do we need to check license and our license book for that or anything?

Reply:
That's a good point. I did not notice jaxb-impl is EDL 1.0 which is on our Caution list. I will clarify if we can use it.
We did include jaxb-impl already in the license book (not for run), but the license changed (from CDDL) with their move to the Eclipse Foundation.

Reply:
I got a little confused here. So the project was moved and now uses a different license. However, the version we are using was licensed under CDDL and is already included in the license book.

Reply:
I have pinged legal about this. Let's wait on their input.

Reply:
Since we are not modifying the dependency and only link to it dynamically we are good.